decidim-initiatives/app/jobs/decidim/initiatives/export_initiatives_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Initiatives
+    class ExportInitiativesJob < ApplicationJob
+      queue_as :default
+
+      def perform(user, format)
+        export_data = Decidim::Exporters.find_exporter(format).new(collection, serializer).export
+
+        ExportMailer.export(user, "initiatives", export_data).deliver_now
```

Comment:
```suggestion
        ExportMailer.export(user, "initiatives", export_data).deliver_later
```
It would be good to use `deliver_later` as it's being used now to send emails